<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureWgt</class>
 <widget class="QMainWindow" name="CaptureWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video capture</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="view"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="capture"/>
   <addaction name="separator"/>
   <addaction name="resolution"/>
   <addaction name="settings"/>
   <addaction name="laser"/>
   <addaction name="flipY"/>
   <addaction name="flipX"/>
   <addaction name="brightnessRange"/>
   <addaction name="separator"/>
   <addaction name="pixmap"/>
  </widget>
  <widget class="QDockWidget" name="brightnessDw">
   <property name="minimumSize">
    <size>
     <width>48</width>
     <height>128</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>48</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="capture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/arrow.png</normaloff>:/images/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </action>
  <action name="settings">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/wrench.png</normaloff>:/images/wrench.png</iconset>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
  </action>
  <action name="laser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/laser.png</normaloff>:/images/laser.png</iconset>
   </property>
   <property name="text">
    <string>laser</string>
   </property>
   <property name="toolTip">
    <string>Feature emphasis</string>
   </property>
  </action>
  <action name="pixmap">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/pixmap.png</normaloff>:/images/pixmap.png</iconset>
   </property>
   <property name="text">
    <string>pixmap</string>
   </property>
   <property name="toolTip">
    <string>Save current image as pixmap</string>
   </property>
  </action>
  <action name="brightnessRange">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/brightness.png</normaloff>:/images/brightness.png</iconset>
   </property>
   <property name="text">
    <string>Brightness range</string>
   </property>
   <property name="toolTip">
    <string>Brightness range</string>
   </property>
  </action>
  <action name="flipY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/flipY.png</normaloff>:/images/flipY.png</iconset>
   </property>
   <property name="text">
    <string>Flip Y</string>
   </property>
   <property name="toolTip">
    <string>Flip an image around Y axis</string>
   </property>
  </action>
  <action name="resolution">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/resolution.png</normaloff>:/images/resolution.png</iconset>
   </property>
   <property name="text">
    <string>Video resolution</string>
   </property>
   <property name="toolTip">
    <string>Preferred video resolutions list</string>
   </property>
  </action>
  <action name="flipX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/flipX.png</normaloff>:/images/flipX.png</iconset>
   </property>
   <property name="text">
    <string>Flip X</string>
   </property>
   <property name="toolTip">
    <string>Flip an image around X axis</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
